core/src/main/scala/org/apache/spark/sql/execution/CoprocessorRDD.scala

```diff
@@ -371,7 +371,7 @@ case class RegionTaskExec(child: SparkPlan,
 
       val schemaInferer: SchemaInfer = if (satisfyDowngradeThreshold) {
         // Should downgrade to full table scan for one region
-        logger.info(
+        logger.debug(
```

Comment:
Such short info is ok and it's useful.

Reply:
how about setting its log level to warn?